l10n_fr_siret/models/res_partner.py

```diff
@@ -0,0 +1,167 @@
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+logger = logging.getLogger(__name__)
+try:
+    from stdnum.fr import siren, siret
+except ImportError:
+    logger.debug("Cannot import stdnum")
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    siren = fields.Char(
+        string="SIREN",
+        size=9,
+        tracking=50,
+        help="The SIREN number is the official identity "
+        "number of the company in France. It composes "
+        "the first 9 digits of the SIRET number.",
+    )
```

Comment:
Hi @fredericgrall did you see https://github.com/odoo/odoo/blob/18.0/addons/l10n_fr/models/res_partner.py#L10 ?

Is there an incidence ?

Reply:
@bealdav,I'll take a look. Thanks for your comment.

Reply:
@bealdav, i'm look the siret field is overide for add inverse methode